src/actions/index.js

```diff
@@ -27,6 +27,10 @@ import {
   dragColumnDivider,
   startDragColumnDivider,
   stopDragColumnDivider,
+  dragOutputDivider,
+  startDragOutputDivider,
+  stopDragOutputDivider,
```

Comment:
Hate to keep doing this to you but it looks like we’ve it another “rule of three” here—three different cases where we want to be doing basically the same thing (handling the dragging of a divider and updating some flex bases accordingly). So, I think we should have a single set of actions to handle divider dragging, and just store the flex bases in a map with keys like `editors`, `columns`, and `output`.

Reply:
Yeah I was wondering about that. Can update this.